### PR TITLE
JSAB2-86 My ranking page

### PR DESCRIPTION
**Pull request by kyya**
+ add a ranking item and my ranking item

### CODE REVIEW

**Comment by kyya** [comment]
It looks like this.
but if you want to see the real effect,
please modify some code in the Leaderboard,
like this.
```jsx
import React from 'react';
import MyRanking from './MyRanking';

function Leaderboard() {
  return <MyRanking />;
}

export default Leaderboard;
```
image

File: frontend/src/Leaderboard/MyRankingItem.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  gold: PropTypes.number,
+  kingdom: PropTypes.number,
+};
+
+MyRankingItem.defaultProps = {
+  ranking: 1,
+  gold: 0,
+  kingdom: 0,
+};
+
```
Comment (GuJialu) [inline frontend/src/Leaderboard/MyRankingItem.jsx:88]: Why is here an extra empty line?

**Review by GuJialu — COMMENTED** [review]
Please remove extra empty line

File: frontend/src/Leaderboard/MyRankingItem.jsx (hunk 1 above)
Comment (kyya) [reply frontend/src/Leaderboard/MyRankingItem.jsx:88]: It has been fixed now.

**Review by GuJialu — APPROVED** [review]

**Review by HanqingZ — APPROVED** [review]

**Review by GuJialu — APPROVED** [review]

**Review by HONGTAO-T — APPROVED** [review]

**Review by evasimonyi — APPROVED** [review]

**Pull request merged**